transaction/core/src/validation/validate.rs

```diff
@@ -1330,22 +1334,25 @@ mod tests {
         }
     }
 
+    // FIXME: This test needs to involve a Tx with more than one output to make
+    // sense
```

Comment:
👍

Reply:
Maybe worth capturing in a testing fast-follow-on ticket?

Reply:
Maybe we should also consider what tests should be blocking for moving forward with the release?

Reply:
Yea i chatted with James and Eugene about this offline, they are planning a fix to land after this PR

IMO we should change the `transaction_core_test_utils::get_transaction` thing so that the test transaction is creates has a change output as well, since it's better test coverage, all of the real life transactions have a change output i think

Reply:
this is now #1866